calculator.py

```diff
@@ -11,13 +11,15 @@ def create_parser():
 def take_expresion(file_name) -> str:
     try: 
         with open(file_name,'r') as f:
-            return f.read()
+            lines = f.read().splitlines()
```

Comment:
You can read the file line by line, instead of reading the whole file, and splitting the input.